.github/workflows/trigger-release-candidate-build-and-deploy.yml

```diff
@@ -28,9 +28,30 @@ jobs:
 
           echo "Latest commit on 'staging' branch is: $(git log -1 --format='%H' staging)"
 
-          echo "Latest commit on 'main' branch is: $(git log -1 --format='%H' main)"
+          # echo "Latest commit on 'main' branch is: $(git log -1 --format='%H' main)"
+          echo "Latest commit on 'psridharan/trigger-stage-deploy' branch is: $(git log -1 --format='%H' psridharan/trigger-stage-deploy)"
 
-          echo "About to merge 'main' branch into 'staging' branch"
-          git merge --verbose main -m "Merging main branch into staging branch"
+          # echo "About to merge 'main' branch into 'staging' branch"
+          echo "About to merge 'psridharan/trigger-stage-deploy' branch into 'staging' branch"
+          # git merge --verbose main -m "Merging main branch into staging branch"
+          git merge --verbose psridharan/trigger-stage-deploy -m "Merging psridharan/trigger-stage-deploy branch into staging branch"
 
           git push origin staging
+      - name: Trigger repository_dispatch to build images
```

Comment:
Should add comment here explaining why repository_dispatch is necessary (https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow)

Reply:
Maybe also link out to https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch for quick reference.

Reply:
@atolopko-czi - will do!